web/themes/custom/cybersecurity/cybersecurity.theme

```diff
@@ -34,16 +34,21 @@ function tailwindcss_get_active_theme() {
  * Changing the publication field label for Publication search page.
  */
 function cybersecurity_preprocess_field(&$variables) {
-
-  // Check if the field label in the correct view mode and content type.
-  if ($variables['element']['#bundle'] == 'publication' && $variables['element']['#view_mode'] == 'search_result_with_snippet') {
-    $variables['label'] = t('Published') . ': ';
+  $ctypes = ['publication', 'news'];
```

Comment:
try to avoid abbreviations in general, using $content_types for example spells out exactly what the intent is.